shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -309,6 +321,31 @@ void AccessibilityBridge::AddSemanticsNodeUpdate(
     PrintNodeSizeError(nodes.back().node_id());
   }
 
+  // Handles root node update.
+  float new_view_pixel_ratio = delegate_.GetViewPixelRatio();
```

Comment:
Why not just pass this into AddSemanticsNodeUpdate directly from the PlatformView (or test)?  That is the only place AddSemanticsNodeUpdate is ever called, so it avoids the extra delegate callback and associated complexity